content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+
+
+
+## Traffic Management
+
+
+- **Improved** resolution of headless services via in-agent DNS to include endpoints
+from other clusters that are on the same network.
+  ([Issue #27342](https://github.com/istio/istio/issues/27342))
+
+- **Improved** usage of AUTO_PASSTHROUGH Gateways to no longer require configuring the `ROUTER_MODE` option on the gateway deployment; instead, it is automatically detected
+  ([Issue #33127](https://github.com/istio/istio/issues/33127))
+
+- **Improved** the XDS configuration generation to cache Envoy clusters, substantially improving performance of the control plane in many cases
+  ([Issue #26732](https://github.com/istio/istio/issues/26732))
+
+- **Improved** CNI network plugin to send logs to the CNI daemonset. This allows viewing CNI logs using `kubectl logs`, instead of looking at kubelet logs.
+  ([Issue #32437](https://github.com/istio/istio/issues/32437))
+
+- **Improved** service conflict resolution to favor Kubernetes Services over ServiceEntries with the same hostname.
+
+- **Updated** CNI install container and race condition repair container are combined into one container.
+  ([Issue #33712](https://github.com/istio/istio/issues/33712))
+
+- **Updated** the Istiod debug interface to be only accessible over localhost or with proper authenciation (mTLS or JWT).
+The recommended way to access the debug interface is through `istioctl experimental internal-debug`, which handles
+this automatically.
+  
+
+- **Added** a flag setting ShutdownDuration in pilot and its default value is 10s.
+  
+
+- **Added** an environment variable `PILOT_STATUS_UPDATE_INTERVAL` that is the interval to update the XDS distribution status and its default value is `500ms`.
+  
+
+- **Added** The Istio sidecar agent will offer HTTP endpoint localhost:15009/debug/<typeurl>.  GET requests
+to that URL will be resolved by sending an xDS discovery "event" to istiod.  This can be disabled by setting
+the following in the Istio Operator: meshConfig.defaultConfig.proxyMetadata.PROXY_XDS_DEBUG_VIA_AGENT=false
+  ([Issue #22274](https://github.com/istio/istio/issues/22274))
+
+- **Added** support for overriding the locality of the WorkloadGroup template in
+auto registered WorkloadEntries. Locality overrides can be passed in through
+Envoy bootstrap configuration.
+  ([Issue #33426](https://github.com/istio/istio/pull/33426)),([Issue #33426](https://github.com/istio/istio/issues/33426))
+
+- **Added** new metric for tracking distribution of configuration resource sizes being pushed by istiod.
+  ([Issue #31772](https://github.com/istio/istio/issues/31772))
+
+- **Added** experimental support for the Kubernetes Multi-Cluster Services (MCS) host (`clusterset.local`).
+This feature is off by default, but can be enabled by setting the following flags in Istio:
+`ENABLE_MCS_HOST` and `ENABLE_MCS_SERVICE_DISCOVERY`. When enabled Istio will include the MCS host as a
```

Comment:
@nmittler should we remove the `ENABLE_MCS_SERVICE_DISCOVERY` here, since you covered it in the next bullet?

Reply:
@nmittler Bump

Reply:
I would leave it as-is ... that way each section is self-contained.